### PR TITLE
containerized my resume app

### PR DESCRIPTION
**Pull request by mubarak-bunza**
have containerized my resume app using best practices.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM node:15.12.0-alpine
+ENV NODE_ENV dev
+
+# Add a work directory
+WORKDIR /app
+
+# Cache and Install dependencies
```
Comment (startnow65) [inline Dockerfile:7]: Nice

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+RUN npm install
+
+# Copy app files
+COPY . .
+
+# Expose port
+EXPOSE 3000
+
+# Start the app
+CMD ["npm", "start"]
```
Comment (startnow65) [inline Dockerfile:18]: Let's use nginx as the webserver. You can read how it is beneficial to do so [here](https://medium.com/intrinsic-blog/why-should-i-use-a-reverse-proxy-if-node-js-is-production-ready-5a079408b2ca).

To do this, you'll need to set up a [multi-staged docker build](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#use-multi-stage-builds)

File: Dockerfile (hunk 1 above)
Comment (mubarak-bunza) [reply Dockerfile:7]: Thank you.

File: Dockerfile (hunk 2 above)
Comment (mubarak-bunza) [reply Dockerfile:18]: Done.

**Comment by startnow65** [comment]
Let's make things a bit more secure, update the prod Dockerfile so that the resulting image runs as non-root

**Comment by mubarak-bunza** [comment]
> 

created new_guest user to run commands as non-root.

**Pull request merged**